tests/end-to-end/backup-restore-test/backupe2e/namespacecontroller.go

```diff
@@ -59,29 +74,18 @@ func (n namespaceControllerTest) labelTestNamespace(namespaceName string) error
 	namespaceCopy := namespace.DeepCopy()
 	namespaceCopy.SetLabels(labels)
 
-	_, err = n.coreClient.CoreV1().Namespaces().Update(namespaceCopy)
+	_, err = n.coreInterface.CoreV1().Namespaces().Update(namespaceCopy)
 	return err
 }
 
-func (n namespaceControllerTest) waitForResourceQuota(namespaceName string) error {
-	timeout := time.After(5 * time.Minute)
-	tick := time.Tick(1 * time.Second)
-
-	var messages string
-
-	for {
-		select {
-		case <-tick:
-			_, err := n.coreClient.CoreV1().ResourceQuotas(namespaceName).Get(resourceQuotaObjName, metav1.GetOptions{})
-			if err != nil {
-				messages += fmt.Sprintf("%+v\n", err)
-				continue
-			}
-
-			return nil
-
-		case <-timeout:
-			return fmt.Errorf("unable to fetch resourcequota:\n %v", messages)
+func (n NamespaceControllerTest) waitForResourceQuota(namespaceName string) error {
+	return retry.Do(func() error {
```

Comment:
what would be the timeout here  ?

Reply:
retry.Do has default timeout 100 ms with doubling backoff and does 10 attempts which gives around 50 s timeout

Reply:
I forgot that delay here is set to 500 ms. It means that whole timeout will be around 250 s